cellfinder_napari/detect.py

```diff
@@ -170,6 +172,14 @@ def widget(
         worker.returned.connect(
             lambda points: add_layers(points, viewer=viewer)
         )
+
+        def update_progress_bar(*props):
+            # props is a (label, max, value) tuple
```

Comment:
docstring? also typing?

Reply:
I've expanded the arguments and added typing. I think that makes a docstring redundant?

Reply:
to me a docstring is more about explaining what the function does rather than the inputs, but I guess it's fairly obvious here